test.sh

```diff
@@ -3,10 +3,22 @@
 cd ./BEIMA.Backend.Test
 # Run backend unit tests
 dotnet test ./BEIMA.Backend.Test.csproj || { echo "Unit tests failed"; exit 1; }
-# Change back to root directory
+# Navigate to backend project
+cd ../
+cd ./BEIMA.Backend
+# Start the backend API locally
+func start BEIMA.Backend.csproj --csharp &
+# Switch to FT directory
+cd ../
```

Comment:
I'm running the backend tests on my machine, and I found if you cd into BEIMA.Backend and run `dotnet test` from there it will run both of our backend test projects. We could use that instead and save a few lines of code here